tutorials/foreign-object.html

```diff
@@ -0,0 +1,813 @@
+        &lt;foreignObject @selector="foreignObject"&gt;
+            &lt;div @selector="outer"
+                xmlns="http://www.w3.org/1999/xhtml"
+                class="outer"
+            &gt;
+                &lt;div @selector="inner" class="inner"&gt;
+                    &lt;form @selector="form" id="form"&gt;
+                        &lt;input @selector="name" type="text" id="name" name="name"/&gt;
+                        &lt;input @selector="submit" type="submit" value="submit"/&gt;
+                    &lt;/form&gt;
+                &lt;/div&gt;
+            &lt;/div&gt;
+        &lt;/foreignObject&gt;
+    `
+});
+</code></pre>
+
+        <p>
+            As the default styles for our <code>form</code> are a little boring, we will also add the following CSS just to make our elements a 
+            little more presentable. It's also possible to add CSS via an SVG 
+            <a href="https://developer.mozilla.org/en-US/docs/Web/SVG/Element/style" target="_blank"><code>style</code></a> element, or inline 
+            styles via the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/style" target="_blank">style</a> attribute 
+            if you prefer, but as we are adding a substantial amount of CSS, we will use a separate stylesheet.
+        </p>
+
+        <pre><code>.outer {
+    background: linear-gradient(92.05deg, hsl(355, 100%, 68%) 12.09%, hsl(355, 100%, 63%) 42.58%, hsl(355, 85%, 54%) 84.96%);
+    padding: 6px;
+    width: 100%;
+    height: 100%;
+}
+
+.inner {
+    background-color: hsl(210deg 36.67% 11.76%);;
+    width: 100%;
+    height: 100%;
+}
+
+.form {
+    display: flex;
+    font-family: sans-serif;
+    flex-direction: column;
+    justify-content: space-evenly;
+    padding: 0 32px;
+    width: 100%;
+    height: 100%;
+}
+
+input {
+    all: unset;
+    background: hsl(218, 80%, 2%);
+    color: white;
+    cursor: text;
+    font-size: 1rem;
+    height: 56px; 
+    outline-color: hsl(355, 100%, 63%);
+    outline-width: thin;
+    padding: 0 32px;
+}
+
+input::placeholder {
+    color: hsl(0, 0%, 50%);
+}
+
+input:focus-visible {
+    outline-style: inset;
+}
+
+button {
+    background-color: hsl(218, 80%, 2%);
+    border: unset;
+    cursor: pointer;
+    font-family: sans-serif;
+    font-size: 1rem;
+    font-weight: 600;
+    height: 56px;
+    outline-color: hsl(355, 100%, 63%);
+    outline-width: thin;
+}
+
+button:focus-visible {
+    outline-style: inset;
+}
+
+button span {
+    background: linear-gradient(92.05deg, hsl(355, 100%, 68%) 12.09%, hsl(355, 100%, 63%) 42.58%, hsl(355, 85%, 54%) 84.96%);
+    -webkit-background-clip: text;
+    background-clip: text; 
+    -webkit-text-fill-color: hsl(0deg 0% 0% / 0%);
+    font-size: 1.2rem;
+}
+</code></pre>
+
+        <div class="paper" id="paper-foreign-object-prevent-default"></div>
+        <p>JointJS source code: <a href="js/foreign-object.js" target="_blank">foreign-object.js</a></p>
```

Comment:
it is extremely difficult to see the difference between the dark-gray background and the black input field / button, which makes it difficult to understand that the user can interact with them. Can we make one of them much lighter? Or can we at least add a red outline to the input field / button?